client/state/action-types.js

```diff
@@ -42,6 +42,7 @@ export const POSTS_REQUEST_SUCCESS = 'POSTS_REQUEST_SUCCESS';
 export const PUBLICIZE_CONNECTIONS_RECEIVE = 'PUBLICIZE_CONNECTIONS_RECEIVE';
 export const PUBLICIZE_CONNECTIONS_REQUEST = 'PUBLICIZE_CONNECTIONS_REQUEST';
 export const PUBLICIZE_CONNECTIONS_REQUEST_FAILURE = 'PUBLICIZE_CONNECTIONS_REQUEST_FAILURE';
+export const READER_FULLPOST_SET_VISIBILITY = 'READER_FULLPOST_SET_VISIBILITY';
```

Comment:
I think it's essentially the same, since we're not setting the state directly but calling one of the action creators.

The splitting might be better for readability, though, I'll change that.